src/routes/Start.svelte

```diff
@@ -11,7 +11,7 @@
 
 <Header>ATAG Report Tool</Header>
 
-<p style="font-size: 1.5em; margin-top: -.25em">Manually check authoring tools against the <a href="https://www.w3.org/TR/ATAG20/">Authoring Tool Accessibility Guidelines</a> (ATAG) and report on the results.</p>
+<p style="font-size: 1.5em; margin-top: -.25em">This tool guides you through the individual requirements of the <a href="https://www.w3.org/WAI/standards-guidelines/atag/">W3C Authoring Tool Accessibility Guidelines (ATAG)</a>, to help you record your evaluation of authoring tools in consistent reports.</p>
```

Comment:
1. The tool doesn't do any check, not manual nor automatic.
2. Also better to link to ATAG Overview page in this intro
3. Some clarifications that I think are quite important

Reply:
Question: why are we using a custom style/font-size rather than the "overview box" of the WAI site design?

Reply:
1. My wording does not say the tool will do the check, it allows the user to check things. The advantage of my original wording is that it is shorter and I believe clearer, but happy to take your suggestion here.
2. Great suggestion
3. OK

Using the [overview box](https://wai-website-theme.netlify.com/components/boxes/) is a great idea, had forgotten about that one, but feel it is perfect for what I was trying to achieve here.

Reply:
Latest version of this Pull Request now uses the box, links to ATAG Overview page and has your clarifications,